--- sympa-6.2.76/src/sbin/archived.pl.in	2024-12-16 05:46:08.000000000 -0500
+++ sympa-6.2.76/src/sbin/archived.pl.in	2026-03-28 13:24:47.502568249 -0400
@@ -27,11 +27,11 @@
 
 use lib split(/:/, $ENV{SYMPALIB} || ''), '--modulesdir--';
 use strict;
 use warnings;
 use English qw(-no_match_vars);
-use Getopt::Long;
+use Getopt::Long qw(:config no_ignore_case);
 use Pod::Usage;
 use POSIX qw();
 
 use Conf;
 use Sympa::Constants;
--- sympa-6.2.76/src/sbin/bounced.pl.in	2024-12-16 05:46:08.000000000 -0500
+++ sympa-6.2.76/src/sbin/bounced.pl.in	2026-03-28 13:24:36.216439987 -0400
@@ -33,11 +33,11 @@
 
 use lib split(/:/, $ENV{SYMPALIB} || ''), '--modulesdir--';
 use strict;
 use warnings;
 use English qw(-no_match_vars);
-use Getopt::Long;
+use Getopt::Long qw(:config no_ignore_case);
 use Pod::Usage;
 use POSIX qw();
 
 use Conf;
 use Sympa::Constants;
--- sympa-6.2.76/src/sbin/bulk.pl.in	2024-12-16 05:46:08.000000000 -0500
+++ sympa-6.2.76/src/sbin/bulk.pl.in	2026-03-28 13:24:27.054332270 -0400
@@ -28,11 +28,11 @@
 
 use lib split(/:/, $ENV{SYMPALIB} || ''), '--modulesdir--';
 use strict;
 use warnings;
 use English qw(-no_match_vars);
-use Getopt::Long;
+use Getopt::Long qw(:config no_ignore_case);
 use Pod::Usage;
 use POSIX qw();
 
 use Conf;
 use Sympa::Constants;
--- sympa-6.2.76/src/sbin/task_manager.pl.in	2024-12-16 05:46:08.000000000 -0500
+++ sympa-6.2.76/src/sbin/task_manager.pl.in	2026-03-28 13:24:54.296604487 -0400
@@ -28,11 +28,11 @@
 
 use lib split(/:/, $ENV{SYMPALIB} || ''), '--modulesdir--';
 use strict;
 use warnings;
 use English qw(-no_match_vars);
-use Getopt::Long;
+use Getopt::Long qw(:config no_ignore_case);
 use Pod::Usage;
 use POSIX qw();
 
 use Conf;
 use Sympa::Constants;
